src/constants/common.ts

```diff
@@ -151,3 +151,5 @@ export enum HashType {
 
 export const MAINNET_URL = `https://${config.BASE_URL}`
 export const TESTNET_URL = `https://${ChainName.Testnet}.${config.BASE_URL}`
+
+export const typeIdCodeHash = '0x00000000000000000000000000000000000000000000000000545950455f4944'
```

Comment:
Usually a constant will be defined in UPPERCASE, e.g. `TYPE_ID_CODE_HASH`

Ref: https://github.com/airbnb/javascript/#naming--uppercase